kolibri/content/constants/presets.py

```diff
@@ -0,0 +1,29 @@
+"""
+This module contains constants representing the ids of FormatPreset.
+"""
+from django.utils.translation import ugettext_lazy as _
+
+# constants for FormatPreset
+HIGH_RES_VIDEO = "high_res_video"
+MID_RES_VIDEO = "mid_res_video"
+LOW_RES_VIDEO = "low_res_video"
+VECTOR_VIDEO = "vertor_video"
+
+EN_SUBTITLE = "en_subtitle"
+ES_SUBTITLE = "es_subtitle"
+ZH_SUBTITLE = "zh_subtitle"
```

Comment:
Sorry for my slow following, but i am still not quite sure if I understand the issue here. Are you saying some spanish videos have Spanish captions that are translated from English caption, and users who can hear and understand Spanish found the re-translated Spanish caption not correct? And marking the original language for videos is for translators who translate vedios according to caption rather than audio?